data/tlg1701/tlg002/tlg1701.tlg002.1st1K-grc1.xml

```diff
@@ -0,0 +1,635 @@
+                <title>The Testament of Abraham Vol. 2</title>
+                <author xml:lang="lat">Testamentum Abrahamae</author>
+                <editor>Montague Rhodes James</editor>
+                <sponsor>HARVARD COLLEGE LIBRARY</sponsor>
+                <funder>Harvard Library Arcadia Fund</funder>
+                <principal>Gregory Crane</principal>
+                <respStmt>
+                    <persName xml:id="DDD">Digital Divide Data</persName>
+                    <resp>Corrected and encoded the text</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Gregory Crane</persName>
+                    <resp>Editor-in-Chief, Perseus Digital Library</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Matt Munson</persName>
+                    <resp>Project Manager (University of Leipzig), 2016 - present</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Annette Gessner</persName>
+                    <resp>Project Assistant (University of Leipzig) 2015 - present</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Thibault Clérice</persName>
+                    <resp>Lead Developer (University of Leipzig)</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Bruce Robertson</persName>
+                    <resp>Technical Advisor</resp>
+                </respStmt>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Harvard College Library</authority>
+                <idno type="filename">tlg1701.tlg002.1st1K-grc1.xml</idno>
+                <availability>
+                    <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International
+                        License</p>
+                </availability>
+                <date>2016</date>
+                <publisher>Harvard College Library</publisher>
+                <pubPlace>United State</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+                            <title>The Testament of Abraham</title>
+                            <editor>
+                                <persName>
+                                    <name>Montague Rhodes James</name>
+                                </persName>
+                            </editor>
+                            <author ref="urn:cts:greekLit:tlg1701" xml:lang="lat">Testamentum Abrahamae</author>
+                            <imprint>
+                                <publisher>Cambridge University Press</publisher>
+                                <pubPlace>Cambridge</pubPlace>
+                                <date>1892</date>
+                            </imprint>
+                            <biblScope unit="volume">2</biblScope>
```

Comment:
This is all in one single volume, so the volume number should be "1" here as well.